models/actions/tut-transformer/main.rb

```diff
@@ -0,0 +1,8 @@
+require 'models/profiles/rocks'
+module Tutorials
+    module Actions
+        class Main < Roby::Actions::Interface
```

Comment:
You don't need this anymore. This is basically done with the Robot.actions block in the robot's config file now.